Source/Core/Core/Config/GraphicsSettings.cpp

```diff
@@ -84,9 +84,9 @@ const Info<bool> GFX_WAIT_FOR_SHADERS_BEFORE_STARTING{
     {System::GFX, "Settings", "WaitForShadersBeforeStarting"}, false};
 const Info<ShaderCompilationMode> GFX_SHADER_COMPILATION_MODE{
     {System::GFX, "Settings", "ShaderCompilationMode"}, ShaderCompilationMode::Synchronous};
-const Info<int> GFX_SHADER_COMPILER_THREADS{{System::GFX, "Settings", "ShaderCompilerThreads"}, 1};
+const Info<int> GFX_SHADER_COMPILER_THREADS{{System::GFX, "Settings", "ShaderCompilerThreads"}, -1};
```

Comment:
Let's leave this one alone and only change the precompiler threads, as doing async compile on multiple threads might cause *more* stutters or general performance degradation if a game generates a lot of new shaders at once.

Reply:
Agreed, this would likely require more precise investigation as to if and when using more than one thread there will be beneficial and how good the current heuristic of (# logical CPUs - 3) is.